### ISSUE TITLE
Unable to add new bubble to the existing ones and update the UI accordingly

### ISSUE THREAD

**Issue by pravingaikwad07** [igalata/Bubble-Picker#25]
Firstly, Thanks for this library. Its very usefull.

I want to add new bubbles to the existing ones. Is it possible to do so ? If yes, then how to do it.

**Comment by igalata** [comment]
Hi @pravingaikwad123 
Thanks! There is no such feature now, but it will be implemented soon

**Comment by think-twice-code-once** [comment]
Does it need long time to implement this feature ? I really appreciate if you implement it soon :(. Or any trick ? I tried to clear the list and add all new items but it doesn't work.

**Comment by tehfonsi** [comment]
This feature would be really nice, I guess most of the time you want to add bubbles as you receive them with an API call!

**Comment by igalata** [comment]
The temporary solution is described here #14 
I have no much time to work on this library regularly, so the updates appear not really often. But of course the functionality is needed and will be implemented soon :)

**Comment by tehfonsi** [comment]
Thanks for your response! The temporary solution only works if you load items once, my case is to show some items on start and then add more items later. Is there a temporary solution for this or can you guide me in the right direction?

**Comment by igalata** [comment]
No, I think I need to implement this feature in such way that if you add more bubbles, they will just appear near the others without animation interruption. I'll start working on it

**Comment by tehfonsi** [comment]
Great, thank you! Looking forward to it!

**Comment by shriharsha-bhagwat** [comment]
Any updates on this issues?

**Comment by wintermoon9** [comment]
Any updates? This is definitely needed. 👍
